modules/cudf/src/data_frame.ts

```diff
@@ -1494,6 +1494,22 @@ export class DataFrame<T extends TypeMap = any> {
       {} as SeriesMap<{[P in keyof T]: Bool8}>));
   }
 
+  sum(subset?: (keyof T)[], dtype?: DataType) {
+    subset     = (subset == undefined) ? this.names as (keyof T)[] : subset;
+    const temp = new Table({columns: this.select(subset)._accessor.columns});
+
+    let common_dtype: DataType = temp.getColumnByIndex(0).type;
+    const sums: number[]         = new Array(temp.numColumns);
+    for (let i = 0; i < temp.numColumns; ++i) {
+      const col    = temp.getColumnByIndex(i);
+      common_dtype = findCommonType(common_dtype, col.type);
+      sums[i]      = Number(col.sum());
```

Comment:
In the python notebook you are able to sum `StringSeries`, however, currently, our `sum` implementation for a series is only available on `NumericSeries`. I'm assuming we want to change this? I ask because the binding is marked as complete.